sources/packages/backend/apps/db-migrations/src/sql/Student/Add-col-disability-status.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS disability_status sims.disability_status;
+
+COMMENT ON COLUMN sims.students.disability_status IS 'Disability status of the student.';
```

Comment:
IMO, we can about the PD and PPD in the comment, it will help the future developer, when they look into the db